backend/src/api/app.ts

```diff
@@ -62,7 +61,33 @@ authenticatedRoute.use(async (req, res, next) => {
     return res.status(401).send('Not logged in');
   }
   return next();
-});
+};
+
+const checkUserSignedTerms = (req, res, next) => {
+  if (!req.requestContext.authorizer.dateAcceptedTerms) {
```

Comment:
We should create a common utility check function that also checks `        (user.acceptedTermsVersion &&  user.acceptedTermsVersion !== getToUVersion(user as AuthUser))` and use this function wherever in the codebase we check `user.dateAcceptedTerms`.

Reply:
I'm fine not being extremely strict on the server side for now with TOU.